it/uk/gov/hmrc/agentsubscriptionfrontend/controllers/BusinessIdentificationControllerISpec.scala

```diff
@@ -530,7 +503,7 @@ trait BusinessIdentificationControllerISpec extends BaseISpec with SessionDataMi
       val result = await(controller.showConfirmBusinessForm(request))
 
       result should containSubstrings(
-        routes.BusinessIdentificationController.submitBusinessDetailsForm(Some("sole_trader")).url)
+        routes.BusinessIdentificationController.submitBusinessDetailsForm(IdentifyBusinessType.SoleTrader).url)
     }
 
     "redirect to the Check Business Type  page if there is no KnownFactsResult in session because the user has returned to a bookmark" in {
```

Comment:
Want an extra test for `showConfirmBusinessForm` when there is no "businessType" in the session.